components/atoms/Logo/Logo.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'routes';
+
+const LogoImage = styled.img`
+  width: 50px;
+`;
+
+const Logo = () => {
+  return (
+    <Link route="/">
+      <a>
+        <LogoImage src={`${process.env.ASSET_HOST}/images/logo.png`} alt="logo" />
```

Comment:
It would be better to use SVG file for logo (like a React component)

Reply:
Ok, that was an example of using the prefix